.github/CONTRIBUTING.md

```diff
@@ -8,12 +8,12 @@ get involved:
   Select `New issue`, fill in a descriptive title, and provide as much detail 
   as is needed for the issue to be reproduced.
 
-* **Notebooks**: If you would like to contribute a notebook, please 
-  create a [fork](https://help.github.com/articles/fork-a-repo/) of the repository 
-  from the `master` branch and create a 
+* **Notebooks**: If you would like to contribute a notebook, please do it in [Qiskit Community Tutorials](https://github.com/qiskit-community/qiskit-community-tutorials).
```

Comment:
This one sentence seems sufficient to me unless @nonhermitian thinks otherwise. The rest of the older text in here about forking the repo and submitting a pull request is already in the contributing docs for the `qiskit-community-tutorials` repo, so I don't think we need to duplicate that here.

If new notebooks are accepted in _this_ repo then @nonhermitian probably needs to help with the wording since I personally don't know about that process.